testing/cpm/cpm_cccl-version-2-7.cmake

```
@@ -32,8 +32,8 @@ file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/override.json
       "version": "2.7.0",
       "git_shallow": false,
       "git_url": "https://github.com/NVIDIA/cccl.git",
-      "git_tag": "e5229f2c7509ced5a830be4ae884d9e1639e8951"
-    }
+      "git_tag": "v2.7.0-rc2"
+    },
```

Unlike Python, JSON is invalid with extra commas at the end..
```suggestion
    }
```